src/components/Chat.js

```diff
@@ -20,7 +20,7 @@ const StyledInput = styled(TextArea)`
   height:1.5em;
   line-height:1.5em;
   min-width 0;
-  width:100%;
+  width:86%;
```

Comment:
Width percentages is not the correct way to style controls to appear side by side.
If we make the Chat frame wider the send button will grow?  thats not what we want...

Use [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) or floats....

Reply:
The send button needs to have a fixed size and the textarea needs to take everything else